src/run_nps_wbm.R

```diff
@@ -0,0 +1,680 @@
+run_nps_wbm <- function(prepped_data,
+                        name = "site_no",
+                        PET_Method = c("Oudin"),
+                        hock_coef = 4,
+                        Snowpack.Init = 0, # according to Mike's code
+                        Soil.Init = 0, # Init full from Mike's code, empty in Ambers
+                        Shade.Coeff = 1,
+                        T.Base = 0) {
+
+  # # Code for testing:
+  # prepped_data <- prep_for_wbm(read_csv("data/all/CollectionofHydSaxe2021/EASTERN TEMPERATE FORESTS_010010202002.csv") %>% mutate(loc = paste(x, "_", y)), name = "loc")
+  # PET_Method <- "Oudin"
+  # hock_coef <-  4
+  # Snowpack.Init <- 0   # according to Mike's code
+  # Soil.Init <- 0 # Init full from Mike's code, empty in Ambers
+  # Shade.Coeff <- 1
+  # T.Base <- 0
+  # name = "loc"
+
+  climate_data <- prepped_data[[1]]
+
+  points2 <- prepped_data[[2]]
+
+  # Load WBM FUnctions
+
+  ##############################################################################
+  ## WB_functions.R
```

Comment:
See above comment - WB_functions should be moved to prep function to avoid re-loading (time sink).  :D